gamebook/game_manager.py

```diff
@@ -39,10 +39,15 @@ def run_scene(self):
         if len(self.current_scene.options) == 0:
             # if the current scene has no options, the game ends
             self.output_instance.exit("you reached the end - game over")
+            return ""
 
         user_input = self.input_instance.\
             ask_for_user_inputs(self.current_scene.options)
 
+        if user_input == "quit":
+            self.output_instance.exit("exiting...")
```

Comment:
Maybe try to move all of your strings (especially those sent to the user into a messages.py file so it will be easy to change them in one place?)

Reply:
good idea :+1: